pkg/logs/config/endpoints.go

```diff
@@ -108,6 +108,7 @@ type Endpoints struct {
 	BatchMaxConcurrentSend int
 	BatchMaxSize           int
 	BatchMaxContentSize    int
+	InputChanSize          int
```

Comment:
The new field probably needs to be set in `NewEndpoints` and other constructors.